src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length < 1) {
+    return {};
+  }
+
+  const sourceElements = sourceString.split(';');
+
+  const sourceObject = sourceElements
+    .map((el) => el.trim())
+    .filter((el) => el.length > 0)
+    .reduce((obj, el) => {
+      const key = el.slice(0, el.indexOf(':')).trim();
+      const value = `${el.slice(el.indexOf(':') + 1, el.length).trim()}`;
+
+      return {
+        ...obj,
+        [key]: value,
+      };
+    }, {});
+
+  return sourceObject;
```

Comment:
The chaining of methods should have the same indentation to improve readability. Align your chained methods under each other.